cmake/LittleTypeLibraryConfig.cmake

```diff
@@ -0,0 +1 @@
+include("${CMAKE_CURRENT_LIST_DIR}/LittleTypeLibraryTargets.cmake")
```

Comment:
Does LittleTypeLibraryTargets.cmake exists?

Reply:
Yes, it's created by `install(EXPORT X ...)`, where if you don't define `FILE` arguments then `X` is used for the filename, resulting in `X.cmake`.

This is also what the `Config.cmake` has to `include`.

Reply:
I am going to try to understand the install thing later :), thanks :)